_pages/contact.html

```diff
@@ -29,10 +29,10 @@ <h5>Emails</h5>
                 <td>
                     <h5>Social</h5>
                     <ul>
-                        <li><a href="https://facebook.com/robotics1418">Facebook</a></li>
```

Comment:
Are we not using the facebook page anymore?

Reply:
Yep it's pretty dead so Mr. Knight said to remove it

Reply:
I get removing it from the header, but not listing it at all under contact seems strange

Reply:
Just added @BobbyMi11er to the Facebook as an admin.